### PR TITLE
Add failure talk from AgileandBeyond

### PR DESCRIPTION
**Pull request by stevenjackson**

### CODE REVIEW

**Comment by searls** [comment]
Is this good to merge? I don't know why deploy failed :/ ¯\_(ツ)_/¯

**Comment by stevenjackson** [comment]
Last several PR deploys have failed.  I'm happy with the post (after fixing a link), not sure if anyone else wants to review.

File: app/posts/2017-07-06-looking-for-failure.md [hunk 1]
```diff
@@ -0,0 +1,1447 @@
...
+  url: "https://twitter.com/stevejxsn"
+video:
+  url: "https://player.vimeo.com/video/217241625"
+  type: "vimeo"
+reddit: false
+---
+
+Failures are interesting. They tend to teach lasting personal lessons, but we
+are loathe to share them. The solutions that allow us to overcome failure after
+several attempts become deeply internalized but difficult to share why they
```
Comment (samjonester) [inline app/posts/2017-07-06-looking-for-failure.md]: `but difficult to share why` doesn't read very well. Maybe split this into two sentences?

File: app/posts/2017-07-06-looking-for-failure.md [hunk 2]
```diff
@@ -0,0 +1,1447 @@
...
+matter. We need to get comfortable talking about failure. Unfortunately our
+brains are wired to screw this up. When confronted with a failure, we are
+conditioned to ignore it, make ourselves blameless, or create elaborate
+strategies to avoid the possibility of failure altogether. I’d like to present
+another alternative: embrace failure. It will happen. That’s ok.
+
+How can we challenge our natural human bias? I've spent the last couple years
+digging into "pop" psychology and thinking about how books like "Thinking Fast
+and Slow", "Antifragile", "Drive", "Blink", and "Outliers" apply in a software
+development environment. I'm most excited about this idea that if we can make
```
Comment (samjonester) [inline app/posts/2017-07-06-looking-for-failure.md]: ~this idea~ -> the idea ?

**Pull request merged**